projects/bbb_canbus_listener/main.cpp

```diff
@@ -331,7 +335,7 @@ int main(int argc, char **argv) {
                 break;
             }
             case BMS_FRAME_ID_1: {
-                std::cout << "bms 1:" << std::endl;
+                std::cout << "Received BS1 Frame:" << std::endl;
```

Comment:
BS1 -> BMS1

Reply:
Minor but if not changed then it's also okay

Reply:
Was it really a typo though lol

Reply:
Looks good. Good job